apphub/image_classification/cifar10_fast/cifar10_fast.ipynb

```diff
@@ -457,7 +457,7 @@
     "                         network=network,\n",
     "                         epochs=epochs,\n",
     "                         traces=traces,\n",
-    "                         max_steps_per_epoch=max_steps_per_epoch)\n",
+    "                         max_train_steps_per_epoch=max_train_steps_per_epoch)\n",
```

Comment:
Did you intentionally leave max_eval_steps_per_epoch=max_eval_steps_per_epoch?

Reply:
done